src/main/groovy/com/pszymczyk/consul/ConsulStarterBuilder.groovy

```diff
@@ -44,8 +43,13 @@ class ConsulStarterBuilder {
         this
     }
 
-    ConsulStarterBuilder withPortsConfigFile(File file) {
-        this.portsConfigFile = file
+    ConsulStarterBuilder withConfigDir(Path file) {
```

Comment:
Actually it's not file now, I think that would be better name for this parameter configDirectory